src/components/svg/Svg.css

```diff
@@ -0,0 +1,9 @@
+.svgWrapper {
+    & svg {
+        width: inherit;
```

Comment:
I think the width and height should be removed because we should allow the svg to occupy its default size.

Reply:
👍